.gitignore

```diff
@@ -0,0 +1,29 @@
+*.py[co]
+
+# Packages
+*.egg
+*.egg-info
+dist
+build
+eggs
+parts
+bin
+var
+sdist
+develop-eggs
+.installed.cfg
+
+# Installer logs
+pip-log.txt
+
+#shared objects
+*.so
+
+#config
+cf_units/etc/site.cfg
+
+# Created by editiors
```

Comment:
I wish we could find the origin of "editiors" and correct it so it doesn't keep getting propagated :flushed: